_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -1,5 +1,4 @@
 //MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
-//Model dictionary trimmed on: 31-01-2017 23:17 (UTC)
```

Comment:
can you please reinsert this line using whatever is on the latest master for metastation?